wiki/Chat_Console/en.md

```diff
@@ -3,21 +3,21 @@ Basic Chat Console
-| <Pale>   | Non-supporters                                                                                                                                                                                           |
-| <Yellow> | ![](Supporter_tag.png "fig:Supporter_tag.png") Supporters                                                                                                                                                |
-| <Red>    | [Global Moderation Team](Global_Moderation_Team "wikilink") or [Quality Assurance Team](Quality_Assurance_Team "wikilink") member                                                                        |
-| <Green>  | Line contains your name or certain keywords that you have added to trigger a "highlight". A copy of the message will also appear in a dedicated channel, \#highlight, which contains all of these lines. |
-| <Blue>   | Private Message                                                                                                                                                                                          |
-| <Cyan>   | [peppy](peppy "wikilink"), the creator of osu!.                                                                                                                                                          |
-| <Pink>   | [BanchoBot](BanchoBot "wikilink")                                                                                                                                                                        |
+| Colour | Who?                                                                                                                                                                                                     |
+|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| White  | You                                                                                                                                                                                                      |
+| Pale   | Non-supporters                                                                                                                                                                                           |
+| Yellow | ![Supporter Tag](/wiki/shared/osu!supporter.png "Supporter Tag") Supporters                                                                                                                                            |
+| Red    | [Global Moderation Team](/wiki/People/Global_Moderation_Team/en.md "Global Moderation Team") or [Quality Assurance Team](/wiki/People/Quality_Assurance_Team/en.md "Quality Assurance Team") member      |
```

Comment:
Replace all instance of `/*.md` (en.md, de.md, any with .md extension).

The osu-web will do the proper direction by itself. 

So, for this line:-
```Markdown
[Global Moderation Team](/wiki/People/Global_Moderation_Team/ "Global Moderation Team") or [Quality Assurance Team](/wiki/People/Quality_Assurance_Team/ "Quality Assurance Team") member
```
(see the missing `en.md`?)